### PR TITLE
Create hourly run jobs for customer1 and create separated input files for them

### PR DESCRIPTION
**Pull request by xiyuxie-google**
Input is based on https://github.com/QiliangCui/bm-infra/pull/44 from b/427646581 and https://github.com/QiliangCui/bm-infra/pull/7 from b/429118042

Adding CUSTOMER1_ as prefix to current RunType and will use this as filter for dashboard, while keeping information about backend. RunType spanner column has been update to string(256)

### CODE REVIEW

File: cases/autotune_torchax_jax_customer1.csv [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Device,Model,MaxNumSeqs,MaxNumBatchedTokens,TensorParallelSize,MaxModelLen,Dataset,InputLen,OutputLen
+v6e-4,google/gemma-3-12b-it,64,64,8,1280,sonnet,1000,100
```
Comment (QiliangCui) [inline cases/autotune_torchax_jax_customer1.csv:2]: if it is v6e-4, the TensorParallelSize can't be 8. it should be 4, too.

File: scripts/scheduler/hourly_run.sh [hunk 2]
```diff
@@ -34,11 +34,17 @@ export REPO_MAP="$REPO_MAP_STRING"
 echo "./scripts/scheduler/create_job.sh ./cases/hourly.csv \"\" $TAG HOURLY"
 ./scripts/scheduler/create_job.sh ./cases/hourly.csv "" $TAG HOURLY
 
+echo "./scripts/scheduler/create_job.sh ./cases/hourly_customer1.csv \"\" $TAG CUSTOMER1_HOURLY"
```
Comment (QiliangCui) [inline scripts/scheduler/hourly_run.sh:37]: HOURLY_CUSTOMER1

File: scripts/scheduler/hourly_run.sh [hunk 3]
```diff
@@ -34,11 +34,17 @@ export REPO_MAP="$REPO_MAP_STRING"
...
+./scripts/scheduler/create_job.sh ./cases/hourly_customer1.csv "" $TAG CUSTOMER1_HOURLY
```
Comment (QiliangCui) [inline scripts/scheduler/hourly_run.sh:38]: HOURLY_CUSTOMER1

File: scripts/scheduler/hourly_run.sh [hunk 4]
```diff
@@ -34,11 +34,17 @@ export REPO_MAP="$REPO_MAP_STRING"
...
+
 # Run gpu_1 on even hours, gpu_2 on odd hours
 # Because I don't have enough h100-8 now.
 if (( 10#$HOUR_NOW % 2 == 0 )); then
   echo "./scripts/scheduler/create_job.sh ./cases/hourly_gpu_1.csv \"\" $TAG HOURLY"
   ./scripts/scheduler/create_job.sh ./cases/hourly_gpu_1.csv "" $TAG HOURLY
+
+  echo "./scripts/scheduler/create_job.sh ./cases/hourly_gpu_customer1.csv \"\" $TAG CUSTOMER1_HOURLY"
+  ./scripts/scheduler/create_job.sh ./cases/hourly_gpu_customer1.csv "" $TAG CUSTOMER1_HOURLY
```
Comment (QiliangCui) [inline scripts/scheduler/hourly_run.sh:47]: HOURLY_CUSTOMER1

File: scripts/scheduler/hourly_run.sh [hunk 5]
```diff
@@ -77,11 +83,17 @@ echo "./scripts/scheduler/create_job.sh ./cases/hourly_jax.csv \"\" $TAG HOURLY_
 echo "./scripts/scheduler/create_job.sh ./cases/hourly_torchax_jax.csv \"\" $TAG HOURLY_AX_JAX TPU_COMMONS \"TPU_BACKEND_TYPE=jax;MODEL_IMPL_TYPE=vllm\""
 ./scripts/scheduler/create_job.sh ./cases/hourly_torchax_jax.csv "" $TAG HOURLY_AX_JAX TPU_COMMONS "TPU_BACKEND_TYPE=jax;MODEL_IMPL_TYPE=vllm"
 
+echo "./scripts/scheduler/create_job.sh ./cases/hourly_torchax_jax_customer1.csv \"\" $TAG CUSTOMER1_HOURLY_AX_JAX TPU_COMMONS \"TPU_BACKEND_TYPE=jax;MODEL_IMPL_TYPE=vllm\""
+./scripts/scheduler/create_job.sh ./cases/hourly_torchax_jax_customer1.csv "" $TAG CUSTOMER1_HOURLY_AX_JAX TPU_COMMONS "TPU_BACKEND_TYPE=jax;MODEL_IMPL_TYPE=vllm"
```
Comment (QiliangCui) [inline scripts/scheduler/hourly_run.sh:87]: CUSTOMER1_HOURLY_AX_JAX -> HOURLY_AX_JAX_C1
The RunType has a length <= 16 limitation.

File: scripts/scheduler/hourly_run.sh (hunk 2 above)
Comment (QiliangCui) [reply scripts/scheduler/hourly_run.sh:37]: or HOURLY_C1

File: scripts/scheduler/hourly_run.sh [hunk 6]
```diff
@@ -77,11 +83,17 @@ echo "./scripts/scheduler/create_job.sh ./cases/hourly_jax.csv \"\" $TAG HOURLY_
...
+
 if [[ "$HOUR_NOW" == "00" || "$HOUR_NOW" == "12" ]]; then
   # vLLM
   echo "./scripts/scheduler/create_job.sh ./cases/autotune.csv \"\" $TAG AUTOTUNE"
   ./scripts/scheduler/create_job.sh ./cases/autotune.csv "" $TAG AUTOTUNE
 
+  echo "./scripts/scheduler/create_job.sh ./cases/autotune_customer1.csv \"\" $TAG CUSTOMER1_AUTOTUNE"
+  ./scripts/scheduler/create_job.sh ./cases/autotune_customer1.csv "" $TAG CUSTOMER1_AUTOTUNE
```
Comment (QiliangCui) [inline scripts/scheduler/hourly_run.sh:95]: AUTOTUNE_C1

File: scripts/scheduler/hourly_run.sh (hunk 3 above)
Comment (xiyuxie-google) [reply scripts/scheduler/hourly_run.sh:38]: Same as previous

File: scripts/scheduler/hourly_run.sh (hunk 2 above)
Comment (QiliangCui) [reply scripts/scheduler/hourly_run.sh:37]: i see. thank you!

**Review by QiliangCui — APPROVED** [review]
